src/accelerate/accelerator.py

```diff
@@ -179,9 +179,10 @@ class Accelerator:
             the execution on one process only.
         dataloader_config (`DataLoaderConfiguration`, *optional*):
             A configuration for how the dataloaders should be handled in distributed scenarios.
-        deepspeed_plugin ([`~utils.DeepSpeedPlugin`], *optional*):
+        deepspeed_plugin ([`~utils.DeepSpeedPlugin`] or list of [`~utils.DeepSpeedPlugin`], *optional*):
```

Comment:
This should now also mention the dict possibility (same with the type annotation below). I would also like to see a bit more explanation on what happens if there is more than one plugin. I think, ideally, once this is properly added to the accelerate docs, a reference could be added here instead of inflating the docstring, as I imagine most users don't need that option.